README.md

```diff
@@ -1 +1,20 @@
 # 📢 Pyxpp
+![photo_2024-10-25_13-54-08(1)](https://github.com/user-attachments/assets/4ae8e6f4-de0d-45ce-9492-a35cb8802e24)
+
+## 🗿 Main information
+  ### :computer: How it works
+   The user inputs their code using c++. Translator takes the input code, creates an input.py file, and runs the python program. The python program generates an output.cpp file containing the translated C++ code. 
```

Comment:
> The user inputs their code using c++

The user enters the path to the input file with the extension .py in the "pyxpp" software, which is a package written in python.

Reply:
> creates an input.py file

It doesn't create a .py file, it opens it.

Reply:
> python program

Specify explicitly that this is a translator.

Reply:
> translated C++

Still, this is Python translated to C++, not the other way around.